packages/typography/src/elements/CodeBlock.tsx

```diff
@@ -36,32 +70,44 @@ export const CodeBlock = React.forwardRef<HTMLPreElement, ICodeBlockProps>(
       _size = 'lg';
     }
 
+    useEffect(() => {
+      addEventListener('resize', updateContainerTabIndex);
+      updateContainerTabIndex();
+
+      return () => {
+        removeEventListener('resize', updateContainerTabIndex);
```

Comment:
Do we also need to cancel any pending denounces here as well? [debounce.cancel()](https://lodash.com/docs/4.17.15#debounce)